pepper-apis/src/main/java/org/broadinstitute/ddp/json/FileUploadUrlResponse.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.ddp.json;
+
+import com.google.gson.annotations.SerializedName;
+
+public class FileUploadUrlResponse {
+    @SerializedName("url")
+    String url;
+
+    @SerializedName("guid")
+    String guid;
+
+    @SerializedName("httpMethod")
```

Comment:
If this is being used to indicate a resumable URL, let's expose this as a boolean `resumable` property instead of the HTTP method name. A developer working with this API should be aware, and familiar with the documentation surrounding Google's bucket upload APIs. The method used here is defined by Google's API and we should not be embedding that into ours in the event their API changes.

Reply:
Exposed `resumable`